pnpm-lock.yaml

```diff
@@ -1,4 +1,4 @@
-lockfileVersion: '6.1'
+lockfileVersion: '6.0'
```

Comment:
pnpm is opaque to me... do we care about this downgrade?

Reply:
don't worry, a random PR from me will upgrade it soon anyway ☠️ , and the cycle will continue

Reply:
I can't find the exact link but I came across this in another thread recently and the tldr was that it just flapped based on a local pnpm version. Not majorly important